### PR TITLE
DOCS-8199 Add Continuous Testing and CI/CD Integrations Documentation for Bitrise

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Previews the changes in https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/pull/27 and https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/28.

When ready to merge, don't forget to update the `pull_config.yaml` with:

```
      - repo_name: synthetics-test-automation-bitrise-step-upload-application
        contents:
        - action: pull-and-push-file
          branch: main
          globs:
          - 'README.md'
          options:
            dest_path: '/continuous_testing/cicd_integrations/'
            file_name: 'bitrise_upload.md'
            front_matters:
              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/blob/main/README.md"]
              title: Continuous Testing and Bitrise
              kind: documentation
      
      - repo_name: synthetics-test-automation-bitrise-step-run-tests
        contents:
        - action: pull-and-push-file
          branch: main
          globs:
          - 'README.md'
          options:
            dest_path: '/continuous_testing/cicd_integrations/'
            file_name: 'bitrise_run.md'
            front_matters:
              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/blob/main/README.md"]
              title: Continuous Testing and Bitrise
              kind: documentation
```

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -289,6 +289,35 @@
...
+        contents:
+        - action: pull-and-push-file
+          branch: alai97/readme-doc-review
+          globs:
+          - 'README.md'
+          options:
+            dest_path: '/continuous_testing/cicd_integrations/'
+            file_name: 'bitrise_run.md'
+            front_matters:
+              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/blob/alai97/readme-doc-review/README.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:316]:
```suggestion
              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/blob/main/README.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -289,6 +289,35 @@
...
+            file_name: 'bitrise_upload.md'
+            front_matters:
+              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/blob/alai97/readme-doc-review/README.md"]
+              title: Continuous Testing and Bitrise
+              kind: documentation
+      
+      - repo_name: synthetics-test-automation-bitrise-step-run-tests
+        contents:
+        - action: pull-and-push-file
+          branch: alai97/readme-doc-review
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:309]:
```suggestion
          branch: main
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -289,6 +289,35 @@
...
+          globs:
+          - 'README.md'
+          options:
+            dest_path: '/continuous_testing/cicd_integrations/'
+            file_name: 'bitrise_upload.md'
+            front_matters:
+              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/blob/alai97/readme-doc-review/README.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:302]:
```suggestion
              dependencies: ["https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/blob/main/README.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 4]
```diff
@@ -289,6 +289,35 @@
               aliases:
               - /serverless/serverless_integrations/plugin
 
+      - repo_name: synthetics-test-automation-bitrise-step-upload-application
+        contents:
+        - action: pull-and-push-file
+          branch: alai97/readme-doc-review
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:295]:
```suggestion
          branch: main
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by rtrieu — APPROVED** [review]
Hey @alai97, this all looks good to me. One thing to consider would be to ask Design to create a version of the bitrise logo cards to say "run" and "upload", because as it is, users don't know what they're clicking into unless they hover over the card, and it looks like it's an accidental dupe.

Screenshot 2024-06-14 at 1 38 57 PM

**Pull request merged**